### PR TITLE
fix: update CSS selectors, display extension icon and menu

### PR DESCRIPTION
**Pull request by phoz0n**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+JUST TRYING TO MAKE IT WORK AGAIN
+
+SEEMS TO BE WORKING AS I AM WRITING THIS IN JUNE 2024
+
+MADE THE OPTIONS.HTML CLICKABLE ON THE EXTENSION
+
+A DROPDOWN OPENS WHERE YOU CAN SELECT uppercase OR LOWERCASE
+
+TEXT-TRANSFORM CAPITALIZE DOES NOT WORK ANYMORE, COULD NOT FIX IT. NEED TO APPLY CAPITALIZE TO FIRST LETTER OF EVERY WORDS ON THE TITLE. STOP.
+
```
Comment (digiz3d) [inline README.md]: issue: pls remove

File: manifest.json [hunk 2]
```diff
@@ -19,7 +19,15 @@
         "run_at": "document_start"
     }],
     "options_ui": {
-        "page": "options.html"
+        "page": "options.html",
+        "browser_style": true
```
Comment (digiz3d) [inline manifest.json]: issue: according to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui#syntax it should not be set

File: options.html [hunk 3]
```diff
@@ -1,18 +1,34 @@
...
-    <meta charset="utf-8">
-  </head>
-  <body>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TOO MUCH CAPS Options</title>
+    <!-- Bootstrap CSS -->
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet">
```
Comment (digiz3d) [inline options.html]: note: heavy for just a selector, can we make it simpler in css?

File: options.html [hunk 4]
```diff
@@ -1,18 +1,34 @@
...
+            <label for="myStyle">Text transform style:</label>
+            <select id="myStyle" name="myStyle" class="form-control">
+                <option id="l" value="lowercase">Lowercase</option>
+                <option id="u" value="uppercase">Uppercase</option>
+            </select>
+        </div>
     </form>
+
+    <!-- Bootstrap JS for modal and other components (if needed) -->
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.bundle.min.js"></script>
```
Comment (digiz3d) [inline options.html]: issue: pls remove this dependency, its too much

File: options.js [hunk 5]
```diff
@@ -8,28 +7,17 @@ function saveOptions(e) {
...
-                sel = "#c"
-                break;
-            case 'uppercase':
-                sel = "#u"
-                break;
-            default:
-                sel = "#l"
-                break;
-        }
-        document.querySelector(sel).setAttribute("selected", "selected");
```
Comment (digiz3d) [inline options.js:23]: praise: nicely done

File: toomuchcaps.js [hunk 6]
```diff
@@ -1,24 +1,34 @@
...
+    // Append style to document element to ensure it applies correctly
+    document.documentElement.appendChild(style);
+}
+
+function onError(error) {
+    console.error(`Error in injecting CSS: ${error}`);
 }
 
-window.addEventListener('load', function () {
+console.log("Content script loaded");
```
Comment (digiz3d) [inline toomuchcaps.js]: todo: remove

File: toomuchcaps.js [hunk 7]
```diff
@@ -1,24 +1,34 @@
...
-    var ytbTextTransformStyle = "lowercase";
-    if (item.ytbTextTransformStyle) {
-        ytbTextTransformStyle = item.ytbTextTransformStyle;
-    }
-    var head = document.head || document.getElementsByTagName('head')[0];
-    var css = '#video-title,\
-    .title.style-scope.ytd-video-primary-info-renderer,\
-    .ytp-title-link.yt-uix-sessionlink.ytp-title-fullerscreen-link,\
-    .miniplayer-title.style-scope.ytd-miniplayer { text-transform: ' + ytbTextTransformStyle + ' !important; }';
+    console.log("Injecting CSS with style:", item.ytbTextTransformStyle);
```
Comment (digiz3d) [inline toomuchcaps.js]: nitpick: remove ?

**Review by digiz3d — CHANGES_REQUESTED** [review]
Nice thank youuuu
Just a few things to fix

**Review by digiz3d — APPROVED** [review]
good, LGTM 👏

File: options.html (hunk 3 above)
Comment (phoz0n) [reply options.html]: Simple design is good resolved.

**Pull request merged**